userfiles/civet_virtual_study/views/_civetqc_legend.html.erb

```diff
@@ -0,0 +1,94 @@
+
```

Comment:
There is no need to duplicate all the files in the view code for the CivetStudy if you inherit from the class...